inspec.yml

```diff
@@ -1,23 +1,28 @@
 name: cis-aws-foundations-baseline
 title: cis-aws-foundations-baseline
 maintainer: MITRE InSpec Team
-copyright: MITRE, 2018
+copyright: MITRE, 2020
 copyright_email: inspec@mitre.org
 license: Apache-2.0
 summary: 'InSpec Profile to validate the CIS AWS Foundations Benchmark v1.1.0'
 version: 1.0.5
 supports:
   - platform: aws
-
 depends:
   - name: inspec-aws
     git: https://github.com/inspec/inspec-aws.git
 
-attributes:
+inputs:
```

Comment:
Remove any input value that are not needed anymore eg `iam_manager_role_name`

Reply:
Removed the following inputs: [commit link](https://github.com/mitre/cis-aws-foundations-baseline/commit/55e2c9b243819ec55af9b31b5edd5a38829a349f)

- iam_manager_role_name
- iam_master_role_name
- iam_manager_user_name
- iam_master_user_name
- iam_manager_policy_name
- iam_master_policy_name
- sns_topics
- sns_subscriptions